src/kit/Select.tsx

```diff
@@ -127,28 +129,54 @@ const Select: React.FC<React.PropsWithChildren<SelectProps>> = forwardRef(functi
   return (
     <div className={classes.join(' ')} ref={divRef}>
       {label && <Label type={LabelTypes.TextOnly}>{label}</Label>}
-      <AntdSelect
-        disabled={disabled || loading}
-        dropdownMatchSelectWidth={dropdownMatchSelectWidth}
-        filterOption={
-          filterOption === undefined ? (searchable ? handleFilter : true) : filterOption
-        }
-        getPopupContainer={getPopupContainer}
-        maxTagCount={maxTagCount}
-        maxTagPlaceholder={maxTagPlaceholder}
-        options={options}
-        ref={inputRef}
-        showSearch={!!onSearch || !!filterOption || searchable}
-        style={{ width }}
-        suffixIcon={!loading ? <Icon name="arrow-down" size="tiny" title="Open" /> : undefined}
-        value={value}
-        onBlur={onBlur}
-        onDropdownVisibleChange={handleDropdownVisibleChange}
-        onFocus={onFocus}
-        onSearch={onSearch}
-        {...passthrough}>
-        {children}
-      </AntdSelect>
+      {options === undefined ? (
```

Comment:
not sure what's the best way to handle this case with `exactOptionalPropertyTypes`. The code def gets redundant with `exactOptionalPropertyTypes`